en-US/firefox/welcome/page2.lang

```diff
@@ -0,0 +1,49 @@
+## NOTE: Demo https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/welcome/2/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/2/
+
+
+# HTML page title
+# "Pocket" is a product brand name and shouldn't be translated
+;Pocket - Save news, videos, stories and more
```

Comment:
a, b, c, and more? An extra comma proceeding `and`?

Reply:
House style guide is not to use a serial comma so this is correct.